components/form/UploadImageInput.tsx

```diff
@@ -54,19 +58,54 @@ const UploadImageInput: React.FC<UploadImageProps> = ({
       const file = e.target.files[0];
 
       if (file) {
-        const imageName = uuidv4();
-        const imagePath = CDN + userId + '/' + imageName;
+        try {
+          if (imageType === 'profile') {
+            const { data: listData, error: listError } = await supabase.storage
+              .from('images')
+              .list(userId, { recursive: true });
```

Comment:
This line is causing a typescript error, although everything seems to be working. It seams like recursive is not a property we can use here, and it seems to work without it. So maybe this is not necessary. Besides these two changes everything looks great. Keep up the good work!

Reply:
Thanks, @nichgalzin - this is redundant indeed. I've deleted now 🙌